actions/create_workroom_test_rule.py

```diff
@@ -51,7 +47,7 @@ def _get_st2_rules_url(base_url):
 
 def _create_distro_rule_meta(distro, branch, action_ref, version_task_index, distro_release=None):
     # last 3 chars is distro short
-    distro_short = distros[len(distros)-3:]
+    distro_short = distro[len(distro)-3:]
```

Comment:
Will this work even now?  (rhel6 & rhel7) or (centos6 & centos7) will get the same shortname?